backend/btrixcloud/profiles.py

```diff
@@ -176,14 +183,16 @@ async def commit_to_profile(
             origins=json["origins"],
             resource=profile_file,
             userid=uuid.UUID(metadata.get("btrix.user")),
-            oid=uuid.UUID(metadata.get("btrix.org")),
+            oid=oid,
             baseid=baseid,
         )
 
         await self.profiles.find_one_and_update(
             {"_id": profile.id}, {"$set": profile.to_dict()}, upsert=True
         )
 
+        await inc_org_bytes_stored(self.orgs, oid, file_size)
```

Comment:
We should error out of `commit_to_profile()` right away if the storage quota is reached.